test/utils.py

```diff
@@ -970,14 +970,15 @@ def is_instance_expelled(admin_api_url, instance_name):
 
 
 @tenacity.retry(stop=tenacity.stop_after_delay(15), wait=tenacity.wait_fixed(1))
-def wait_for_container_start(container, time_start):
+def wait_for_container_start(container, container_message, time_start):
     container_logs = container.logs(since=int(time_start)).decode('utf-8')
-    assert 'entering the event loop' in container_logs
+    assert container_message in container_logs
```

Comment:
I would like `'entering the event loop'` to be default value of `container_message`.